typescript/infra/config/environments/mainnet3/igp.ts

```diff
@@ -36,6 +36,17 @@ export function getOverheadWithOverrides(local: ChainName, remote: ChainName) {
   return overhead;
 }
 
+function getOracleConfigWithOverrides(chain: ChainName) {
+  const oracleConfig = storageGasOracleConfig[chain];
+  if (chain === 'infinityvm') {
+    // For InfinityVM, override all remote chain gas configs to use 0 gas
+    for (const remoteConfig of Object.values(oracleConfig)) {
+      remoteConfig.gasPrice = '0';
```

Comment:
does this mean that we quote nothing when sending to infinityvm right now?

fwiw we just care about Solana <> InfinityVM being free but all the rest would want it to not be free

Reply:
so e.g. eth->infinity would still quote a fee, but infinity->eth would not